.vscode/settings.json

```diff
@@ -4,6 +4,7 @@
     "OPA_EVAL_CAPABILITIES": "${workspacePath}/build/capabilities.json"
   },
   "opa.roots": [
+    "${workspaceFolder}/lab",
```

Comment:
Is this related to the PR?

Reply:
No, thanks! Removed. Although it _is_ rather tricky to get it right with the "checkOnSave" feature when you want it outside of the main bundle but not the whole workspace... I'll guess I'll have to do some git magic to make sure this line remains unseen. I've removed it from the change now anyway.